### PR TITLE
German translation: High Elves

### PR DESCRIPTION
**Pull request by DaSaubazi**

### CODE REVIEW

File: public/games/the-old-world/high-elf-realms.json [hunk 1]
```diff
@@ -2461,7 +2485,7 @@
         {
           "name_en": "Cavalry spears, Shortbows",
           "name_it": "Lancia Leggera, Archi Corti",
-          "name_de": "Cavalry spears, Shortbows",
+          "name_de": "Kavalleriespeer, Kurzbogen",
```
Comment (nthiebes) [inline public/games/the-old-world/high-elf-realms.json:2488]:
```suggestion
          "name_de": "Kavalleriespeere, Kurzbögen",
```

File: public/games/the-old-world/high-elf-realms.json [hunk 2]
```diff
@@ -2020,7 +2037,6 @@
         {
           "name_en": "Shield",
           "name_it": "Scudo",
-          "name_de": "Shield",
```
Comment (nthiebes) [inline public/games/the-old-world/high-elf-realms.json:2023]: 🙃

**Review by nthiebes — CHANGES_REQUESTED** [review]
Thank you! 🙏

File: public/games/the-old-world/high-elf-realms.json (hunk 1 above)
Comment (DaSaubazi) [reply public/games/the-old-world/high-elf-realms.json:2488]: General topic: Should weapons and equipment for units with multiple models be in singular or plural form? I've seen both used, sometimes even mixed within a single unit.

File: public/games/the-old-world/high-elf-realms.json (hunk 2 above)
Comment (DaSaubazi) [reply public/games/the-old-world/high-elf-realms.json:2023]: Shield is a standard equipment for Dragon Princes. There should be no options for shields at all.

File: public/games/the-old-world/high-elf-realms.json (hunk 1 above)
Comment (nthiebes) [reply public/games/the-old-world/high-elf-realms.json:2488]: Plural

Comment (nthiebes) [reply public/games/the-old-world/high-elf-realms.json:2488]: The high elf dataset is a bit messy and got less love than other armies...

File: public/games/the-old-world/high-elf-realms.json (hunk 2 above)
Comment (nthiebes) [reply public/games/the-old-world/high-elf-realms.json:2023]: True, the whole entry needs to be updated. Hand weapons, Lances and Shields should be one equipment option.

File: public/games/the-old-world/high-elf-realms.json (hunk 1 above)
Comment (DaSaubazi) [reply public/games/the-old-world/high-elf-realms.json:2488]: OK. I will review all entries and fix the english version if necessary.

File: public/games/the-old-world/high-elf-realms.json (hunk 2 above)
Comment (DaSaubazi) [reply public/games/the-old-world/high-elf-realms.json:2023]: I will fix it.

**Pull request merged**